src/storage/core.rs

```diff
@@ -109,6 +109,24 @@ impl<K: Key> Storage<K> {
     ///
     /// [`init()`]: struct.Storage.html#method.init
     pub async fn init(&mut self) -> Result<()> {
+        self.init_ext(true).await
+    }
+
+    /// [`init_noactive()`] used to prepare all environment to further work, but unlike `init`
+    /// doesn't set active blob, which means that first write may take time..
+    ///
+    /// Storage works in directory provided to builder. If directory don't exist,
+    /// storage creates it, otherwise tries to init existing storage.
+    /// # Errors
+    /// Returns error in case of failures with IO operations or
+    /// if some of the required params are missed.
+    ///
+    /// [`init_noactive()`]: struct.Storage.html#method.init
+    pub async fn init_noactive(&mut self) -> Result<()> {

```

Comment:
Just thought, maybe name it `init_lazy()`. Similar to rust vars lazy initialization